--- ../src-base/minecraft/net/minecraft/client/gui/GuiOptions.java
+++ ../src-work/minecraft/net/minecraft/client/gui/GuiOptions.java
@@ -141,7 +141,7 @@
             if (p_146284_1_.field_146127_k == 100)
             {
                 this.field_146297_k.field_71474_y.func_74303_b();
-                this.field_146297_k.func_147108_a(new GuiControls(this, this.field_146443_h));
+                this.field_146297_k.func_147108_a(new net.minecraftforge.client.settings.gui.GuiMenuCategories(this, this.field_146443_h));
             }
 
             if (p_146284_1_.field_146127_k == 102)
